### PR TITLE
Fix A8X issue

### PR DESCRIPTION
**Pull request by sasa8810**
Fix that the location of the file to be copied on iPadAir2 is different

### CODE REVIEW

**Comment by asdfugil** [comment]
Don’t remove $no_baseband = 1 conditionals. Set it to 2 in case of A8X and use an extra elif to check for it.

**Comment by sasa8810** [comment]
updated

**Comment by asdfugil** [comment]
There is actually a better way to do it: store the required disk number into a variable like $disk_no, which is
8 for baseband, 7 for no baseband and 6 for A8X
and use that variable in anything that deals with fakefs.

in this way, all but one of the if $no_baseband can be gotten rid of

**Comment by sasa8810** [comment]
This is all I can do as a beginner:(

**Comment by samh06** [comment]
I can give that a try

**Comment by samh06** [comment]
I still have the --A8X in there cause I don't know how to bring that down to detect A8X automatically

**Comment by asdfugil** [comment]
What is the case on cellular A8X?

**Comment by sasa8810** [comment]
I think can with libmobileidevice, but I don’t know how to do that

**Comment by asdfugil** [comment]
nah just use the $cpid variable, it is set to 0x7001 for A8X

**Comment by sasa8810** [comment]
> セルラーA8Xのケースは何ですか？

I don’t have it

**Comment by samh06** [comment]
ill quickly try do that

**Comment by samh06** [comment]
pretty sure I got it @sasa8810 if you would merge my pull request into your fork that would be appreciated

**Comment by samh06** [comment]
it still doesn't consider cellular A8X though, I don't know what the disk number would be for baseband, would assume 7 right?

**Comment by sasa8810** [comment]
> pretty sure I got it @sasa8810 if you would merge my [pull request into your fork](https://github.com/sasa8810/palera1n-fix-iPadAir2-/pull/1) that would be appreciated thanks

I did it

**Comment by samh06** [comment]
sorry @sasa8810 for making another PR just after you merged that one but I just added a consideration for A8X cellular to it as well

File: palera1n.sh [hunk 1]
```diff
@@ -490,8 +491,10 @@ if [ ! -f blobs/"$deviceid"-"$version".shsh2 ]; then
     done
 
     echo "[*] Testing for baseband presence"
-    if [ "$("$dir"/sshpass -p 'alpine' ssh -o StrictHostKeyChecking=no -p2222 root@localhost "/usr/bin/mgask HasBaseband | grep -E 'true|false'")" = "false" ]; then
-        no_baseband=1
+    if ["${cpid}" == *"0x7001"*]; then
+        disk = 6
+    elif [ "$("$dir"/sshpass -p 'alpine' ssh -o StrictHostKeyChecking=no -p2222 root@localhost "/usr/bin/mgask HasBaseband | grep -E 'true|false'")" = "false" ] && [ ! "$disk" == 6]; then
```
Comment (asdfugil) [inline palera1n.sh:496]:
```suggestion
    elif [ "$("$dir"/sshpass -p 'alpine' ssh -o StrictHostKeyChecking=no -p2222 root@localhost "/usr/bin/mgask HasBaseband | grep -E 'true|false'")" = "false" ] && [ ! "$disk" == 6 ]; then
```

Comment (samh06) [reply palera1n.sh:496]: ill add this to my PR to sasa's fork

Comment (samh06) [reply palera1n.sh:496]: already fixed in my PR to sasa's fork

**Comment by samh06** [comment]
continued in #253

**Comment by verygenericname** [comment]
shouldn't be needed anymore

**Pull request closed without merging**